backend/lib/endpoints/posts.js

```diff
@@ -1,11 +1,14 @@
 const httpErrors = require("http-errors");
-const Post = require("../models/Post");
-const Comment = require("../models/Comment");
 
 /*
  * /api/posts
  */
 async function routes(app) {
+  const Post = app.mongo.model("Post");
```

Comment:
This is bad and still don't agree with this change. Why did you change this on Fastify's PR changing the way we interact with mongo? If we are following the rule of "one feature per PR" this shouldn't be changed in the first place without any discussion.

Reply:
This is out of scope of the pull request, was already approved by you and merged to master.
If you want to discuss this change please open another ticket.